examples/notebooks/submodel_cracking_DFN_or_SPM.ipynb

```diff
@@ -0,0 +1,263 @@
+    "c_s_surf_t=solution[\"Negative particle surface concentration\"]\n",
+    "disp_t = solution[\"Negative particle surface displacement [m]\"]\n",
+    "l_cr_n_t = solution[\"Negative particle crack length\"]\n",
+    "dl_cr = solution [\"Negative particle cracking rate\"]\n",
+    "t_all = solution[\"Time [s]\"].entries\n",
+    "x = solution[\"x [m]\"].entries[0:19, 0]\n",
+    "c_s_n = solution['Negative particle concentration']\n",
+    "r_n = solution[\"r_n [m]\"].entries[:, 0, 0]\n",
+    "\n",
+    "# plot\n",
+    "def plot_concentrations(t):\n",
+    "    f, (ax1, ax2, ax3, ax4) = plt.subplots(1, 4 ,figsize=(20,4))\n",
+    "    ax1.plot(x, stress_t_n_surf(t=t,x=x))\n",
+    "    ax1.set_xlabel(r'$x_n$ [m]')\n",
+    "    ax1.set_ylabel('$\\sigma_t/E_n$')\n",
+    "    \n",
+    "    plot_c_n, = ax2.plot(r_n, c_s_n(r=r_n,t=t,x=x[0]))  # can evaluate at arbitrary x (single representative particle)\n",
+    "    ax2.set_ylabel('Negative particle concentration')\n",
+    "    ax2.set_xlabel(r'$r_n$ [m]')\n",
+    "    ax2.set_ylim(0, 1)\n",
+    "    ax2.set_title('Close to current collector')\n",
+    "    ax2.grid()\n",
+    "    \n",
+    "    plot_c_n, = ax3.plot(r_n, c_s_n(r=r_n,t=t,x=x[10]))  # can evaluate at arbitrary x (single representative particle)\n",
+    "    ax3.set_ylabel('Negative particle concentration')\n",
+    "    ax3.set_xlabel(r'$r_n$ [m]')\n",
+    "    ax3.set_ylim(0, 1)  \n",
+    "    ax3.set_title('In the middle')\n",
+    "    ax3.grid()\n",
+    "\n",
+    "    plot_c_n, = ax4.plot(r_n, c_s_n(r=r_n,t=t,x=x[-1]))  # can evaluate at arbitrary x (single representative particle)\n",
+    "    ax4.set_ylabel('Negative particle concentration')\n",
+    "    ax4.set_xlabel(r'$r_n$ [m]')\n",
+    "    ax4.set_ylim(0, 1)  \n",
+    "    ax4.set_title('Close to separator')\n",
+    "    ax4.grid()\n",
+    "    plt.show()\n",
+    "    \n",
+    "import ipywidgets as widgets\n",
+    "widgets.interact(plot_concentrations, t=widgets.FloatSlider(min=0,max=7200,step=10,value=0));"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Plot results using the default functions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "4a14a036a0b24d11960ff6c336645698",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "interactive(children=(FloatSlider(value=0.0, description='t', max=2.0868031269501226, step=0.02086803126950122…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "label = [\"Crack model\"]\n",
+    "output_variables = [\"Negative particle crack length\", \"Positive particle crack length\"]\n",
+    "quick_plot = pybamm.QuickPlot(solution, output_variables, label)\n",
+    "quick_plot.dynamic_plot();"
```

Comment:
I am barely seeing any crack propagation here. Change the parameter values for this example, or add a note to explain that you don't expect much propagation in a single cycle. Might be good to also plot the x-averaged value so we can see it is actually changing, a little bit

Reply:
It doesn't expect much propagation in a single cycle. How to change the y-axis scale in the dynamic plot?

Reply:
With some difficulty. The easiest way is to change the variable limits https://pybamm.readthedocs.io/en/latest/source/plotting/quick_plot.html

Otherwise you can do `quick_plot.fig.get_axes()` to get the matplotlib axes and edit those individually

Reply:
it can use `variable_limits='tight'` to address this issue.